public/app/plugins/datasource/elasticsearch/specs/elastic_response.test.ts

```diff
@@ -42,10 +42,15 @@ describe('ElasticResponse', () => {
 
     it('should return 1 series', () => {
       expect(result.data.length).toBe(1);
-      expect(result.data[0].target).toBe('Count');
-      expect(result.data[0].datapoints.length).toBe(2);
-      expect(result.data[0].datapoints[0][0]).toBe(10);
-      expect(result.data[0].datapoints[0][1]).toBe(1000);
+      expect(result.data[0].name).toBe('Count');
```

Comment:
I'd refactor these tests a bit more by removing `beforeEach()` calls wherever they aren't used more than once. Instead put their contents into the `it()` calls.
Also typing `result` properly where applicable - so e.g. `DataQueryResponse` instead of `any`.

Aside from this, it looks good to me. :+1:

Reply:
@s-h-a-d-o-w done, moved all `beforeEach()` to `it()`.